notebooks/MAST/downloading_transiting_planet_data/downloading_exoplanet_data.ipynb

```diff
@@ -0,0 +1,1287 @@
+{
```

Comment:
<p>I will admit that I got here and was like "wait, it's only 2000 rows?" I did then remember that you explained the "pagesize" parameter above, but it would be good somewhere (here or there) to remind the user that you've purposefully only asked to return the first 2000 rows. Whereas I presume if you did not set that parameter, you would get the same 24815 rows as the astropy table? This is important because otherwise users will miss the existence of data that they actually want/need to see.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='67'/>